### DISCUSSION TITLE
Problems at the training with new models

### DISCUSSION THREAD

**Discussion by nativenoble** [TheLastBen/fast-stable-diffusion#1190] [Q&A] [answered]
Good morning everyone,
I have a problem of understanding, last week I had my pictures trained against the samdoesart model and the results were really great. Now I have tried it against 2 other models and the results are absolutely not good. 
Against the samdoesart model I also have to use my trigger word that my images are taken. In the 2 other attempts, my image is always taken no matter if I use the trigger or not. For example, if I say <trigger> take a selfie with Bruce Willis I have my face in the picture 2 times. What am I currently doing wrong? Has something changed?

**Comment by chewtoys** [comment, marked as answer]
i think you set Text_Encoder_Training_Steps too high, start like with 250, train the model, check it, if its too low add another 50 and so on.
make sure UNet_Training_Steps is set to 0

**Reply by nativenoble** [reply]
thanks for your reply, I have selected the default setting and that is 250 steps, should I set it even lower?

**Reply by chewtoys** [reply]
no idea im just guessing, the scrip was changed yeah, lots of people having problems

I did follow this advice, and it seems to be working
https://github.com/TheLastBen/fast-stable-diffusion/discussions/1162#discussioncomment-4501747

To get a good balance between training time and good results, you need to do some tuning, start with 600 unet at 2e-5, if not enough, resume the training for 200 steps with a slightly lower lr 8e-6 until you get good results, the text encoder should not go over a total of 350 for 10 images during the whole training.

650 2e5 - looks terrible
250
then
150 8e6 - looks way better
0

repeat 8e6 until it look like the model you want, cfg scale 12 to test
